packages/web-components/fast-foundation/src/badge/badge.template.ts

```diff
@@ -5,8 +5,10 @@ export const BadgeTemplate = html<Badge>`
     <template class="${x => (x.circular ? "circular" : "")}">
         <div
             class="badge"
-            style="background-color: var(--badge-fill-${x =>
-                x.fill}); color: var(--badge-color-${x => x.color})"
+            style="${x =>
+                x.fill || x.color
```

Comment:
Is it possible that there's a fill but no color or a color but no fill?

Reply:
Yeah, this isn't ideal. I think we can update but I wanted to mitigate the issue of this being applied when neither was used. I think it's less egregious to have one undefined variable here that's not in use than both. We should fix this though to have both only written if they exist.